projects/vitejs.dev/package.yml

```diff
@@ -1,22 +1,27 @@
-distributable: ~
+distributable: https://github.com/vitejs/vite/archive/refs/tags/v{{version.raw}}.tar.gz
 
 versions:
   github: vitejs/vite/tags
   strip: /v/
 
+dependencies:
+  pnpm.io: "*"
+  nodejs.org: ^18
```

Comment:
this is too strict, what's our strategy @jhheider ? `^16 || ^18 || ^20` or something right?

Reply:
yes... if the package.json defines engines we should mirror it.